daemon/api_validate.go

```diff
@@ -346,19 +353,27 @@ func validationSetForAssert(st *state.State, accountID, name string, sequence in
 	return sets, nil
 }
 
-func validationSetAssertFromDb(st *state.State, accountID, name string, sequence int) (*asserts.ValidationSet, error) {
+func validationSetAssertFromDb(st *state.State, accountID, name string, sequence int) (vset *asserts.ValidationSet, err error) {
```

Comment:
At a first sight, the changes to this function seem to be unrelated from the rest. If it's so, I'd rather have them in a separate commit explaining what changed (or maybe you can expand the text in the current commit, if that's too bothersome).

Reply:
Yes, you right, thanks for spotting. This was unintended change, a result of having a few WIP branches where I take stuff from for this feature. Removed.